graphql/graphql.go

```diff
@@ -811,15 +811,15 @@ type CallData struct {
 // CallResult encapsulates the result of an invocation of the `call` accessor.
 type CallResult struct {
 	data    hexutil.Bytes  // The return data from the call
-	gasUsed hexutil.Uint64 // The amount of gas used
+	gasUsed Long           // The amount of gas used
```

Comment:
Long what?

Reply:
Either a typo or type alias?

Reply:
LONG $ETH

Reply:
long eth, short tron

Reply:
Is this financial advice on core code? @sec

Reply:
Is this financial advice?

Reply:
gas is too long to pass